DDCore/include/DD4hep/ConditionAny.h

```diff
@@ -10,8 +10,8 @@
 // Author     : M.Frank
 //
 //==========================================================================
-#ifndef DD4HEP_CONDITIONSANY_H
-#define DD4HEP_CONDITIONSANY_H
+#ifndef DD4HEP_CONDITIONANY_H
+#define DD4HEP_CONDITIONANY_H
```

Comment:
Why not using `#pragma once` rather than playing with names that will unavoidably collide one day ?